sdk/python/feast/entity.py

```diff
@@ -97,88 +95,68 @@ def __str__(self):
 
     @property
     def name(self) -> str:
-        """
-        Gets the name of this entity.
-        """
         return self._name
 
     @name.setter
-    def name(self, name):
-        """
-        Sets the name of this entity.
-        """
+    def name(self, name: str):
         self._name = name
 
     @property
-    def description(self) -> str:
-        """
-        Gets the description of this entity.
-        """
-        return self._description
+    def value_type(self) -> ValueType:
+        return self._value_type
 
-    @description.setter
-    def description(self, description):
-        """
-        Sets the description of this entity.
```

Comment:
Is there a reason we are removing all comments?

Reply:
as discussed, we will remove all properties